syntax = "proto3";

package payment;

service PaymentService {
  rpc MakePayment (PaymentRequest) returns (PaymentResponse);
}

message PaymentRequest {
  int32 user_id = 1;
  double amount = 2;
  string payment_method = 3;
}

message PaymentResponse {
  string status = 1;
  string message = 2;
}
